README.md

```diff
@@ -48,21 +49,38 @@ or download it
 ```sh
 $ wget https://github.com/scala-exercises/scala-exercises/archive/master.zip
 ```
+##### Configure the database
 
-You'll need a working PostgreSQL 9.4 database and user for running the app. Once the database is running,
-create a user called `scalaexercises_dev_user`
+You'll need a working [PostgreSQL 9.4](http://www.postgresql.org/download/) database and user for running the app. Once the database is running,
+
+- Create a user called `scalaexercises_dev_user`
 
 ```sh
 $ createuser -P -e scalaexercises_dev_user "a_password"
 ```
 
-Create a db called `scalaexercises_dev` and grant all privileges on it to `scalaexercises_dev_user`
+- Create a db called `scalaexercises_dev` and grant all privileges on it to `scalaexercises_dev_user`
 
 ```sh
 $ createdb scalaexercises_dev
 $ psql -c "GRANT ALL PRIVILEGES ON DATABASE scalaexercises_dev TO scalaexercises_dev_user;"
 ```
 
+**Configuring the database in ubuntu**
```

Comment:
I wasn't sure about the platform of the instructions and I decided to keep them. If you are ok with replacing the current instructions I'll do it.